legal/satzung.md

```diff
@@ -0,0 +1,123 @@
+2. Benachrichtigungsfristen gelten als erfüllt, wenn die Benachrichtigung vor Verstreichen der Frist abgeschickt oder veröffentlicht wurde.
+3. Eine Benachrichtigung gilt als zugestellt, wenn sie an die letzte dem Verein bekannte Adresse verschickt oder auf der Webseite des Vereins veröffentlicht wurde.
+4. Bei Zustimmung einer zu benachrichtigenden Person kann gegenüber dieser auf die Einhaltung von Benachrichtigungsfristen und -formalien verzichtet werden.
+
+
+## Mitgliedschaft
+
+1. Mitglieder des Vereins können volljährige natürliche sowie juristische Personen werden.
+2. Die Beitrittserklärung erfolgt gegenüber dem Vorstand.
+3. Über Aufnahme entscheidet der Vorstand. Die Aufnahme erfolgt nach freiem Ermessen.
+    Bei Ablehnung ist er nicht verpflichtet, dem Antragsteller die Gründe hierfür mitzuteilen.
+4. Die Mitgliedschaft endet durch Austritt aus dem Verein, durch Tod, Ausschluss, Streichung von der Mitgliederliste.
+5. Ein Mitglied kann durch Beschluss des Vorstandes von der Mitgliederliste gestrichen werden, wenn es trotz schriftlicher Mahnung mit der Bezahlung von Mitgliedsbeiträgen oder von Umlagen im Rückstand ist.
+    Diese Streichung befreit das Mitglied nicht von der Begleichung rückständiger Beiträge und Umlagen.
+6. Ein Mitglied kann ausgeschlossen werden, wenn es schuldhaft in grober Weise die Interessen des Vereins verletzt. Der Ausschluss erfolgt durch Beschluss [des Vorstandes / der Mitgliederversammlung]: Vor der Beschlussfassung muss dem Mitglied Gelegenheit zur mündlichen oder schriftlichen Stellungnahme gegeben werden. Der Beschluss ist schriftlich zu begründen und dem Mitglied mitzuteilen. Hiergegen kann innerhalb eines Monats nach Zugang des Beschlusses beim Vorstand Widerspruch eingelegt werden. Über diesen Widerspruch entscheidet die nächste Mitgliederversammlung endgültig.
+
+
+## Beiträge
+
+1. Der Verein kann einen Jahresbeitrag und einen Aufnahmebeitrag erheben.
+    Die maximale Höhe dieser Beiträge ist von der Mitgliederversammlung festzulegen.
+2. In begründeten Fällen können Mitglieder von der Beitragspflicht ausgenommen werden.
+3. Näheres regelt die Beitragsordnung.
+
+
+## Mitgliederversammlung
+
+1. Oberstes Organ des Vereins ist die Mitgliederversammlung.
+    Ihrer Beschlussfassung unterliegen alle in dieser Satzung oder per Gesetz vorgesehenen Gegenstände,
+    insbesondere
+    1. Wahl des Vorstandes,
+    2. Wahl der Finanzprüfer*innen,
+    3. Beschlussfassung über den Jahresbericht des Vorstandes,
+    4. Entlastung des Vorstandes,
+    6. Verabschiedung der Beitragsordnung,
+    8. Satzungsänderungen und
+    9. Auflösung des Vereins.
+2. Einmal im Geschäftsjahr soll eine ordentliche Mitgliederversammlung stattfinden.
+3. Mitgliederversammlungen werden vom Vorstand unter Einhaltung einer Frist von vierzehn Tagen unter Vorschlag einer Tagesordnung einberufen.
```

Comment:
TBD:
```suggestion
3. Mitgliederversammlungen werden vom Vorstand unter Einhaltung einer Frist von vierzehn Tagen unter Vorschlag einer Tagesordnung einberufen. Eine Mitgliederversammlung ist auch einzuberufen, wenn mindestens 3 Mitglieder unter Angabe eines Zweckes dies verlangen.
```

Reply:
(@busiing) Wie wird die Tagesordnung erstellt, bzw. wie kann man da als Mitglied Punkte aufnehmen lassen? Vorschlag: Das könnte man in der Geschäftsordnung regeln

Reply:
14 Tage eventuell zu kurz. Und sollen wir in der Geschäftsordnung irgendetwas zur Terminfindung regeln? Vorschlag: siehe oben

Reply:
okay, diese Punkte würden dann von der Geschäftsordnung geregelt. passt für mich.